index.html

```diff
@@ -0,0 +1,430 @@
+          <h2 class="p-title">Services</h2>
+          <div class="p-title-line"></div>
+        </section>
+
+        <p class="l-upper-rows">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+        <p class="l-upper-columns">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+        <p class="l-upper-columns">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+
+        <section class="l-services-list">
+            <figure class="l-services-item">
+            <h3>UI & UX DESIGN</h3>
+              <div class="l-icon-outer">
+                <span class="icon-paint-format"></span>
+              </div>
+              <figcaption>There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</figcaption>
+            </figure>
+            <!-- <div class="l-home-spacer"> </div> -->
+            <figure class="l-services-item">
+            <h3>LABORATORY</h3>
+              <div class="l-icon-outer">
+                <span class="icon-lab"></span>
+              </div>
+              <figcaption>There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</figcaption>
+            </figure>
+            <!-- <div class="l-home-spacer"> </div> -->
+            <figure class="l-services-item">
+            <h3>DEVELOPMENT</h3>
+              <div class="l-icon-outer">
+                <span class="icon-cog"></span>
+              </div>
+              <figcaption>There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</figcaption>
+            </figure>
+            <!-- <div class="l-home-spacer"> </div> -->
+        </section>
+
+        <section class="l-title">
+          <div class="p-title-line_little"></div>
+            <h3 class="p-title">Pricing Tables</h3>
+          <div class="p-title-line_little"></div>
+        </section>
+
+        <section class="l-pricing-tables">
+          <ul class="p-pricing-tables">
+            <li>BASIC</li>
+            <li>$5.99 / MONTH</li>
+            <li>3 Projects</li>
+            <li>2 Users</li>
+            <li>1 Gb user</li>
+            <li>1 Calendar</li>
+            <li>
+              <div class="p-button-holder">
```

Comment:
I would prefer to apply presentational classes that makes the `a` look like a button, versus wrapping this in a `div`.